instrumentation/opentelemetry-instrumentation-starlette/src/opentelemetry/instrumentation/starlette/__init__.py

```diff
@@ -239,7 +239,7 @@ def instrument_app(
             meter_provider,
             schema_url="https://opentelemetry.io/schemas/1.11.0",
         )
-        if not getattr(app, "is_instrumented_by_opentelemetry", False):
+        if not getattr(app, "_is_instrumented_by_opentelemetry", False):
```

Comment:
This needs to match the setting at https://github.com/open-telemetry/opentelemetry-python-contrib/pull/3529/files#diff-a4901479b7a21ad4f5d45a7f3e47b2ada3f9d8ae4ae7f9e68c2001e8b8067611L330

I think the problem becomes double telemetry, but presumably no one uses such an old version of starlette to allow the auto-instrumentation to kick in to trigger it.